style.css

```diff
@@ -467,3 +497,235 @@ button:active {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+  }
+
+  .nav ul li {
+    margin: 20px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .first-page {
+    background-image: url(./image/dec-intro.png);
+    background-position: left;
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 100%;
+    min-height: 100vh;
+    padding: 22px;
+  }
+
+  .headl1 {
+    margin: 5em 18em 0;
+    width: 600px;
+  }
+
+  .card-image img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .mob {
+    display: none;
+  }
+
+  .dec {
+    display: block;
+  }
+
+  .card-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    background: #fff;
+    border-radius: 24px;
+    padding: 1rem;
+    height: 486px;
+    width: 1156px;
+    margin: 0 auto;
+    gap: 3.5em;
+  }
+
+  .two,
+  .four {
+    flex-direction: row-reverse;
+  }
+
+  .about-page {
+    display: flex;
+    flex-direction: row;
+    padding: 2.5em;
+  }
+
+  .about-me-left {
+    width: 40%;
+  }
+
+  .about-me-right {
+    width: 60%;
+  }
+
+  .lang {
+    display: flex;
+  }
+
+  .frame-arro {
+    display: flex;
+    justify-content: flex-end;
+    position: relative;
+    left: -46px;
+  }
+
+  .form-cantact {
+    background-image: url(./image/dec-form.png);
+  }
+}
+
+@media (min-width: 768px) {
+  .toolbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .mylogo {
+    margin-left: 150px;
+  }
+
+  .nav {
+    display: flex;
+    justify-content: flex-end;
+    margin-left: 390px;
+  }
+
+  .nav ul {
+    display: flex;
+  }
+
+  .nav-item {
+    margin-left: 444px;
+  }
+
+  .lang li {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+  }
+
+  .nav ul li {
+    margin: 10px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .first-page {
+    background-image: url(./image/dec-intro.png);
+    background-position: left;
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 100%;
+    min-height: 100vh;
+    padding: 22px;
+  }
+
+  .headl1 {
+    margin: 5em 18em 0;
+    width: 600px;
+  }
+
+  .card-image img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .mob {
+    display: none;
+  }
+
+  .dec {
+    display: block;
+  }
+
+  .card-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    background: #fff;
+    border-radius: 24px;
+    padding: 1rem;
+    height: 486px;
+    width: 1156px;
+    margin: 0 auto;
+    gap: 3.5em;
+  }
+
+  .two,
+  .four {
+    flex-direction: row-reverse;
+  }
+
+  .about-page {
+    display: flex;
+    flex-direction: row;
+    padding: 2.5em;
+  }
+
+  .about-me-left {
+    width: 40%;
+  }
+
+  .about-me-right {
+    width: 60%;
+  }
+
+  .lang {
+    display: flex;
+  }
+
+  .frame-arro {
+    display: flex;
+    justify-content: flex-end;
+    position: relative;
+    left: -46px;
+  }
+
+  .form-cantact {
```

Comment:
- Kindly, your desktop contact heading and forms are a little bit small compared to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848&t=VJop6PK5cSpPL27i-0), Please update to stick the design 👍😁


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/213582466-6f6027d5-4849-4a22-a544-07ff64d17ddc.png" width="600">|<img src="https://user-images.githubusercontent.com/52010789/213582469-3f95c828-38d3-42ca-ae35-eb8fa4bb09cb.png" width="600">|


😎️

🤝️